src/vs/editor/contrib/bracketMatching/browser/bracketMatching.ts

```diff
@@ -251,6 +270,32 @@ export class BracketMatchingController extends Disposable implements IEditorCont
 			this._editor.revealRange(newSelections[0]);
 		}
 	}
+	public replaceBrackets(openingContent: string, closingContent: string, editSource?: string): void {
```

Comment:
Please remove the first two parameters, they always seem to be `''`.

Reply:
Thanks for review! Okay, I will revert it to its original implementation.

I changed implementation in https://github.com/microsoft/vscode/pull/160270/commits/b2bc6c55e121389824b12a46545c9ed7380d66cc as I thought it'd better for infrustructure to have general method instead, so command to replace brackets would be implemented even more cleanly... Sorry, I'll try to not include unnecessary changes until needed next time!

Btw, how this editSource in `executeEdits` is used? Can I see it in some output channel when doing text changes?